packages/adminrouter/extra/src/includes/server/open/master.conf

```diff
@@ -118,6 +118,11 @@ location /dcos-metadata/ui-config.json {
 
 # Group: Exhibitor
 # Description: Exhibitor cluster status (unauthenticated)
+##This needs to be unauthenticated so bootstrap can do "exhibitor wait" without
+##connecting to exhibitor directly and therefore having this endpoint
+##unauthenticated is necessary for Exhibitor TLS. This is done in these PRs:
```

Comment:
"these PRs" implies that there are multiple but there's only one linked. I don't mind the grammar but maybe you're missing one?

Also, that linked PR is merged, is it the right one? How can it be?

Personally, I prefer to link to JIRA issues for upcoming work, but don't mind this.

Reply:
edit: "Also, that linked PR is merged, is it the right one? How can it be?" = I misread